crypto/csharp/CryptoOpFpe.cs

```diff
@@ -29,6 +29,13 @@ static void Main(string[] args)
             string pass = string.Empty;
             ConsoleKeyInfo consoleKeyInfo;
 
+            // FPE Cardinality with Nae fpe Cardinality , Cardinality in integer value as key and the respective block size as value.
```

Comment:
making sample complex to read. I believe this should be removed. In this way, need to handle unicode as well which will make it more complex. Instead create 56 byte random IV directly and add comment to update as per cardinality used.

Reply:
ok let me set it only for card 10